itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/editor/CRCServerEditorTest.java

```diff
@@ -0,0 +1,108 @@
+/******************************************************************************* 
+ * Copyright (c) 2019 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.tools.cdk.ui.bot.test.server.editor;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.eclipse.selectionwizard.NewMenuWizard;
+import org.eclipse.reddeer.eclipse.wst.server.ui.wizard.NewServerWizardPage;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.jboss.tools.cdk.reddeer.core.condition.SystemJobIsRunning;
+import org.jboss.tools.cdk.reddeer.core.label.CDKLabel;
+import org.jboss.tools.cdk.reddeer.core.matcher.JobMatcher;
+import org.jboss.tools.cdk.reddeer.server.ui.CDKServersView;
+import org.jboss.tools.cdk.reddeer.server.ui.editor.CRCServerEditor;
+import org.jboss.tools.cdk.reddeer.server.ui.wizard.NewCRCServerWizardPage;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test case covering CRC Server editor layout and behavior.
+ * @author odockal
+ *
+ */
+public class CRCServerEditorTest extends CDKServerEditorAbstractTest {
+
+	private static final Logger log = Logger.getLogger(CRCServerEditorTest.class);
+	
+	private static String CRC_PATH;
+	
+	@BeforeClass
+	public static void setupCRCerverEditorTest() {
```

Comment:
It is the case when on does not pass binary path via maven property into a test and so the mock binary comes into play. It might make sense when one does run just one type of test (cdk3.2+) for real binary but also runs others (and you do not wanna run multiple VMs on our CI) and so you run mock tests...

Reply:
@jkopriva Answer is ok with you or you still want me to do something with it? :)